src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/Tov.cpp

```diff
@@ -103,10 +103,6 @@ void lindblom_rhs(
       d_log_conformal_factor = -0.25 * d_mass_over_radius;
     }
   } else {
-    // This statement is triggered by one of the test examples,
-    // Test_Tov.cpp: void {anonymous}::test_rueter()
-    // Test_Tov.cpp: void {anonymous}::test_baumgarte_shapiro()
-    // not sure what to do about that
     if constexpr (CoordSystem == TovCoordinates::Schwarzschild) {
       ASSERT(mass_over_radius < .5,
```

Comment:
@isaaclegred why did you add this assert? You changed the `test_baumgarte_shapiro` to use the polytropic scaling relations "to prevent nonsensical behavior". As far as I understand the behavior is perfectly sensical, just in different units!

Reply:
I don't think so. If I remember correctly, the issue you run into is that, while you can solve the TOV equations for such a compact star, it's not in the wrong units (which are set by the EOS) but you have a TOV star that is a BH and so that's not physical. Admittedly, this was long enough ago that Isaac and I talked about it that I don't remember all the details, but it was essentially that, I think.

Reply:
Ok yeah perhaps this change should be reverted.  The difficulty is that this is not just a change of units, because the values of m(h)/r(h) change depending on this choice.  If m/r exceeds 1/2, then r(h) is no longer monotonic, which I do not understand.   I think perhaps the problem is that the global M, and R are invariant under this scaling, but the entire star doesn't just scale up, it is reconfigured.  I'm pretty sure that if you tried to construct TOV volume data with the EoS that was there before, it wouldn't make any sense.  I'm fine with removing this check from here, it probably shouldn't be the TOV solvers responsibility to check this, but I also think it's reasonable to call the TOV star that was in the `test_baumgarte_shapiro`   before nonsensical :).